### PR TITLE
[#47] Implemented minor suggestions

### PR DESCRIPTION
**Pull request by mauriel133**
> 1- When edit is clicked and contributions tab is at the front switch to Overview tab otherwise user does not see things on overview tab that are editable and none of the contributions part is editible.
> 
> 2- Set the State/Province field so that the Unspecified goes away as soon as a user clicks. I ended up with state UnspecifiedUtah until I explicitly deleted the unspecified letters.

### CODE REVIEW

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

**Comment by pkdash** [comment]
@zhiyuli Can you review this?

File: theme/templates/accounts/profile.html [hunk 1]
```diff
@@ -1029,6 +1032,14 @@ <h4 class="modal-title" id="deleteCV">Delete CV</h4>
 
             $("tr[data-type='all']").find(".badge").text(collection["total"]);
 
+
+            // Unspecified goes away as soon as a user clicks.
+            $("input[name='state'").click(function () {
```
Comment (zhiyuli) [inline theme/templates/accounts/profile.html]: @mauriel133 
I think there is a typo. this line missed a right square bracket ] after 'state'.

**Comment by mauriel133** [comment]
@zhiyuli, I fixed the typo now.

**Comment by zhiyuli** [comment]
+1

**Pull request merged**